OpenRA.Game/Server/Server.cs

```diff
@@ -293,6 +295,18 @@ public Server(List<IPEndPoint> endpoints, ServerSettings settings, ModData modDa
 
 						foreach (var t in serverTraits.WithInterface<ITick>())
 							t.Tick(this);
+
+						if (State == ServerState.GameStarted)
+						{
+							foreach (var (playerIndex, scale) in orderBuffer.GetTickScales())
```

Comment:
Would this not benefit from adding an interval between queueing tick scale orders to be sent?

Reply:
OK I guess the `if (now < nextUpdate) yield break;` in `GetTickScales()` will cut us off early, saving us from doing anything, so we should be fine here.